randovania/games/fusion/logic_database/Sector 5 ARC.txt

```diff
@@ -400,61 +404,88 @@ Extra - room_id: [13]
+              Varia Suit
+              Hi-Jump and Damage Runs (Advanced) and Cold Damage ≥ 238
   > Bottom of Room
-      # TODO: dmg
-      Varia Suit
+      Any of the following:
+          # Damage Reqs
+          Varia Suit
+          Damage Runs (Advanced) and Cold Damage ≥ 319
 
 > Door to Groznyj Shaft; Heals? False
   * Layers: default
   * L3 Hatch to Groznyj Shaft/Door to Frozen Tower
   * Extra - door_idx: (49,)
   > Bottom of Room
-      # TODO: dam
-      Varia Suit
+      Any of the following:
+          # Damage Reqs
+          Varia Suit
+          Damage Runs (Advanced) and Cold Damage ≥ 16
 
 > Other to Transmutation Trial; Heals? False
   * Layers: default
   * Tunnel to Transmutation Trial/Other to Frozen Tower
   * Extra - door_idx: (79,)
   > Bottom of Room
-      # TODO: dmg
-      Varia Suit
+      Any of the following:
+          # Damage Reqs
+          Varia Suit
+          Damage Runs (Advanced) and Cold Damage ≥ 17
 
 > Door to Data Room; Heals? False
   * Layers: default
   * L3 Hatch to Data Room/Door to Frozen Tower
   * Extra - door_idx: (90,)
   > Door to Mimic Storage
       All of the following:
-          # TODO: dmg
-          Varia Suit
+          Any of the following:
+              # Damage Reqs
+              Varia Suit
+              Space Jump and Damage Runs (Advanced) and Cold Damage ≥ 59
           # Drop down from ledge above https://youtu.be/lUc4sT4UyQY
           Jump Extends (Expert) and Movement (Hypermode)
   > Bottom of Room
-      # TODO: dmg
-      Varia Suit
+      Any of the following:
+          # Damage Reqs
+          Varia Suit
+          Damage Runs (Advanced) and Cold Damage ≥ 150
 
 > Bottom of Room; Heals? False
   * Layers: default
   > Door to Mimic Storage
       All of the following:
-          # TODO: dmg
-          Varia Suit
+          Any of the following:
+              # Damage Reqs
+              Varia Suit
+              Hi-Jump and Space Jump and Damage Runs (Expert) and Cold Damage ≥ 521
           Any of the following:
               Space Jump or Can Freeze Enemies
               # Wall Jump from right side https://youtu.be/-GGGrkNAvF4
               Jump Extends (Expert) and Wall Jump (Intermediate) and Can Single Walljump
   > Door to Groznyj Shaft
-      # TODO: dmg
-      Varia Suit
+      Any of the following:
+          # Damage Reqs
+          Varia Suit
+          Damage Runs (Advanced) and Cold Damage ≥ 20
   > Other to Transmutation Trial
       All of the following:
           Can Bounce in Ball
-          # TODO: dmg
-          Varia Suit
+          Any of the following:
+              # Damage Reqs
+              Varia Suit
+              Damage Runs (Advanced) and Cold Damage ≥ 48
   > Door to Data Room
-      # TODO: dmg
-      Space Jump and Varia Suit
+      All of the following:
+          Space Jump
+          Any of the following:
+              # Damage Reqs
+              Varia Suit
+              Damage Runs (Expert) and Cold Damage ≥ 692
```

Comment:
is this with or without HJ?

Reply:
this one i forgot to add oops, yes this was done with hijump

Reply:
Didnt you say this was done with hijump?